common/modifiers/babylon_modifiers.txt

```diff
@@ -448,3 +448,272 @@ babylon_greener_grass_modifier = {
+	tax_land_add = 0.05
+	country_prestige_mult = -0.03
+}
+
+janissary_legacy = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_rifle_negative.dds
+	building_mobilization_cost_mult = 0.20
+	military_formation_travel_speed_mult = -0.10
+	country_prestige_mult = -0.03
+}
+
+rampant_piracy = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_coins_negative.dds
+	unit_convoy_requirements_mult = 0.20
+	country_trade_route_competitiveness_mult = -0.10
+	country_prestige_mult = -0.03
+}
+
+kurdish_question = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_negative.dds
+	state_radicals_from_discrimination_mult = 0.05
+	state_turmoil_effects_mult = 0.10
+	country_prestige_mult = -0.03
+}
+
+balkan_disarray = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_negative.dds
+	state_radicals_from_discrimination_mult = 0.10
+	state_turmoil_effects_mult = 0.10
+	country_prestige_mult = -0.03
+}
+
+imbalanced_education = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_documents_negative.dds
+	country_institution_schools_max_investment_add = -1
+	interest_group_ig_intelligentsia_pol_str_mult = -0.10
+	country_prestige_mult = -0.05
+}
+
+internal_strife = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_statue_negative.dds
+	country_legitimacy_headofstate_add = -5
+	interest_group_in_opposition_approval_add = -1
+	state_turmoil_effects_mult = 0.10
+	country_prestige_mult = -0.03
+}
+
+sick_man_tanzimat_bureaucracy = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_documents_positive.dds
+	country_bureaucracy_mult = 0.10
+}
+
+sick_man_tanzimat_economy = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_documents_positive.dds
+	building_group_bg_manufacturing_tax_mult = 0.05
+	country_minting_mult = 0.10
+}
+
+sick_man_tanzimat_army = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_documents_positive.dds
+	building_mobilization_cost_mult = -0.05
+}
+
+sick_man_tanzimat_navy = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_documents_positive.dds
+	building_mobilization_cost_mult = -0.10
+}
+
+ottoman_kurdish_liberal_reward = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_gear_negative.dds
+	country_construction_add = -5
+}
+
+ottoman_kurdish_absolute_reward = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
+	country_authority_add = 200
+}
+
+kurdish_liberal_reward = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_gear_positive.dds
+	state_construction_mult = 0.10
+	state_migration_pull_mult = 0.2
+	country_construction_add = 5
+}
+
+balkan_liberal_peace = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
+	state_radicals_from_discrimination_mult = -0.10
+}
+
+balkan_absolute_peace = {
+	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
+	country_decree_cost_mult = -0.10
+}
+
+sick_man_liberal_education = {
```

Comment:
no education access?

Reply:
Just to make them have a better boost? I suppose I could I just didn't think it necessary as the main goal was to make the other law categories easier to pass by strengthening liberal pops.